index.html

```diff
@@ -1363,14 +1377,21 @@ <h4>Break calibration</h4>
           at runtime when this mitigation is running continuously. Any attempts to recalibrate
           will similarly be mitigated against.
         </p>
-        <div class="note">
+        <aside class="note">
+          This mitigation, if applied too often, can deteriorate the pressure state detection reliability.
```

Comment:
deteriorate is a very strong word, also why is this more or a problem when done often? Not a fan of the "applied"